src/data/markdown/docs/01 guides/02 Using k6/05 Options.md

```diff
@@ -263,6 +264,62 @@ export let options = {
 
 </CodeGroup>
 
+### DNS
+
+> _New in v0.29.0_
+
+This is a composite option that provides control of DNS resolution behavior with
+configuration for cache expiration (TTL), IP selection strategy and IP version
```

Comment:
Should we clarify that this has no relation to the real TTL in the DNS record, and that we're considering a new resolver implementation, or it's not worth mentioning?

Reply:
Yes, I think we should mention that k6 doesn't respect the actual TTL value yet.

Reply:
Resolved by a9189d1?